src/client/wetty/download.ts

```diff
@@ -34,12 +38,23 @@ function onCompleteFile(bufferCharacters: string): void {
     mimeType = 'text/plain';
     fileExt = 'txt';
   }
-  const fileName = `file-${new Date()
-    .toISOString()
-    .split('.')[0]
-    .replace(/-/g, '')
-    .replace('T', '')
-    .replace(/:/g, '')}${fileExt ? `.${fileExt}` : ''}`;
+  let fileName;
+  try {
+    if (typeof fileNameBase64 === "string") {
+      fileName = window.atob(fileNameBase64);
+    }
+  } catch (err) {
+    // Filename wasn't base64-encoded so let's ignore it
```

Comment:
Is there a reason to base64 encode the filename?

Reply:
If it is not encoded/escaped and contains a `:`, using `:` as a separator would not work correctly. Base64-encoding seemed like the easiest option that is guaranteed to work, but there are certainly other options that would work too!